CATALOG.md

```diff
@@ -481,6 +481,19 @@ Result Type|normative
 Suggested Remediation|Ensure CNF statefulsets/replica sets can scale in/out successfully.
 Best Practice Reference|https://connect.redhat.com/sites/default/files/2022-05/Cloud%20Native%20Network%20Function%20Requirements%201-3.pdf Section 5.2
 Exception Process|There is no documented exception process for this.
+#### test-containers-image-tag
+
+Property|Description
+---|---
+Test Case Name|test-containers-image-tag
+Test Case Label|lifecycle-test-containers-image-tag
```

Comment:
Shouldn't this test be in the affiliated-certification suite?

Reply:
we have the image pull policy in the lifecycle so its similar to it

Reply:
Agree. Lifecycle does not seem to be the place for this test.